### PR TITLE
usage docs improvements

### PR DESCRIPTION
**Pull request by arbll**
### What does this PR do

Adds some insight about the various parameters used to send a metric to the documentation.

### Motivation

#70
#69

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,28 +14,36 @@ Go 1.7+ is officially supported. Older versions might work but are not tested.
 
 ## Usage
 
+Start by creating a new client instance with the appropriate configuration options:
+
 ```go
-// Create the client
-c, err := statsd.New("127.0.0.1:8125",
+client, err := statsd.New("127.0.0.1:8125",
```
Comment (grosser) [inline README.md]:
```suggestion
client, err := statsd.New(
    "127.0.0.1:8125",
```

File: README.md [hunk 2]
```diff
@@ -14,28 +14,36 @@ Go 1.7+ is officially supported. Older versions might work but are not tested.
 
 ## Usage
 
+Start by creating a new client instance with the appropriate configuration options:
```
Comment (grosser) [inline README.md]:
```suggestion
Start by creating a new client:
```

File: README.md (hunk 1 above)
Comment (arbll) [reply README.md]: I think it's fine the way it currently is. It's easier to see the separation between the address and the list of options

**Comment by arbll** [comment]
Thanks for the suggestions @grosser!

**Pull request merged**